core/indigo-core/molecule/CDXConstants.h

```diff
@@ -24,6 +24,10 @@
 #endif //  defined( _X86_ ) && !defined( _CS3D )
 #endif // QUADCONST
 
+typedef unsigned short int UINT16;
```

Comment:
it is worth to move this into common/base_c/defs.h

Reply:
Yes it is actually worth, they are rather of common usage than specific to the CDX, at least in their intentions. I have moved all three of them to the common/base_c/defs.h
typedef unsigned short int UINT16;
typedef int INT32;
typedef unsigned int UINT32;